vllm/engine/async_llm_engine.py

```diff
@@ -171,6 +171,7 @@ def process_request_output(self,
 
         if verbose and finished:
             logger.info("Finished request %s.", request_id)
+            logger.info("Metrics: %s", request_output.metrics)
```

Comment:
We can do more complicated computation here but to keep the proposal simple, I just print the raw fields.